src/common/conf_file.cpp

```diff
@@ -143,8 +146,9 @@ int ConfFile::_parse_file(const char *addr, size_t len, const char *filename)
             }
 
             ret = _add_config(s, addr, line_len, filename, line);
-            if (ret < 0)
+            if (ret < 0) {
                 return ret;
+            }
```

Comment:
ditto

Reply:
There was no blank line after the return before.